### ISSUE TITLE
Dropdown Code Sandbox (and possible underlying Dropdown)

### ISSUE THREAD

**Issue by Rolias** [pluralsight/design-system#2018]
In the code sandbox for the Dropdown
https://codesandbox.io/s/3g69r?module=/example.tsx
There are several problems
1. A dependency is missing for `@pluralsight/ps-design-system-screenreaderonly`
2. The code imports `ActionItem` as the name for the default export but then uses `<Dropdown >` in the code. The import should specify the name as Dropdown since this is an example for Dropdown.
3. The `onChange` event handler specifies a signature of `onChange={(evt, value, label)`, which would be convenient,  but alas the label parameter  comes in as undefined. Not sure if the example is wrong or if the underlying `onChange` mechanism has a bug.

**Comment by caseybaggz** [comment]
Fixed. Closing.
